docs/atlascli/command/atlas-clusters-search-nodes.txt

```diff
@@ -0,0 +1,59 @@
+.. _atlas-clusters-search-nodes:
+
+===========================
+atlas clusters search nodes
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Manage Atlas Search nodes for your cluster.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for nodes
```

Comment:
```suggestion
     - Help for nodes
```
Should this be capitalized?

Reply:
I don't think it should be. All other commands are lowercase